zlib-rs/src/deflate.rs

```diff
@@ -4226,4 +4249,19 @@ mod test {
             }
         }
     }
+
+    #[cfg(any(target_arch = "x86_64", target_arch = "aarch64"))]
+    mod _cache_lines {
+        use super::State;
+        // TODO: once zlib-rs Minimum Supported Rust Version >= 1.77, switch to core::mem::offset_of
```

Comment:
we inherit the rust compiler's interpretation of `FIXME` for things that should be fixed at some point in the future, and `TODO` for things that should be fixed before the PR is ready.

Given that this needs a rebase anyway: can you change to `FIXME` here?